src/sentry/sentry_metrics/transactions.py

```diff
@@ -52,3 +53,11 @@ class TransactionStatusTagValue(Enum):
     CANCELLED = "cancelled"
     UNKNOWN = "unknown"
     ABORTED = "aborted"
+
+
+class TransactionSatisfactionTagValue(Enum):
+    """Identifier value for the satisfaction of a transaction."""
+
+    SATISFIED = "satisfied"
+    TOLERATED = "tolerated"
+    FRUSTRATED = "frustrated"
```

Comment:
Tag values in ingestion are [here](https://github.com/getsentry/relay/blob/7d1f750583be495b84f77f597a7c37b591556835/relay-server/src/metrics_extraction/transactions.rs#L94L109).